tests/client-dbus/tests/dbus/pool/test_udev.py

```diff
@@ -121,6 +121,9 @@ def _start_service(self):
         """
 
         if self._service is None:
+            self.assertEqual(UdevAdd._process_exists("stratisd"), None)
```

Comment:
AFAICT: This isn't part of any test. What you're doing here is verifying that the internal data of the test infrastructure matches reality as well as possible. Instead of using the test assertions you should use regular assertions or possibly return an exception. Same is true for ```_stop_service_remove_dm_tables```.